src/components/HomePage/PaginationComponents/Pagination.css

```diff
@@ -0,0 +1,54 @@
+#nav-pagination {
+  position: relative;
+  display: flex;
+  flex: row;
+  justify-content: center;
+  padding-bottom: 50px;
+}
+
+.pagination-item-style.page-item.active button {
+  background-color: #f65261;
+  border-color: #f65261;
+}
+
+#nav-pagination button {
+  width: 40px;
+}
```

Comment:
in SCSS it would be:

```
#nav-pagination {
  position: relative;
  display: flex;
  flex: row;
  justify-content: center;
  padding-bottom: 50px;

      &-style {
          margin-left: 5px;
      }

    button {
       width: 40px;
    }
}
```

Reply:
Thank you. Added SCSS in next MR and fixed it in commit 10558d1d1e5a9dd451890f797bd08d7b40d79eba